wizelineacademy/react-apprenticeship-capstone1
# React apprenticeship: fourth minichallenge

## Description
```
Created a global context to store global variables such as the theme and searchparam
Created a provider to pass the props to all the components
Added and modified some of the tests cases to achieve the 70% of test coverage

to run this aplication: npm start
to test the aplication: npm run test


- 
```
## Type of Change




## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

run npm run test to see all the created tests cases and look if they are relevant and have a good implementation
```
## What is the deliverable?
```
React Apprenticeship: fourth mini challenge
```



## Screenshots and links
```

![image](https://user-images.githubusercontent.com/97203630/149851572-1c73b4c2-5834-4548-9c2c-5db9c59fde7e.png)


```
## Notes
```
In local host a mocker service workers is enable, so all the request are going to be intercepted by this
It just return one video to avoid an overcall of the daily quote on the you tube API
```
## Checklist
- [x ] I added tests to prove that my fix is effective or my deliverable works.
- [ x] I reviewed existing Pull Requests before submitting mine.
- [x ] I have commented my code, particularly in hard-to-understand areas
